### ISSUE TITLE
[Issue]: No Keyboard with TooUI App Search

### ISSUE THREAD

**Issue by srykho22** [SamSprung/SamSprung-TooIME#34]
### Issue Description

No keyboard appears when trying to search for an app with TooUI. This makes it difficult to find apps, especially since they are seemingly reshuffled randomly and icons disappear, only reappearing when launched with the phone open. 

Sort of a double-sided issue concerning both TooUI and ToolME (both Google Play latest version)

### Logcat Output

```shell
--------- beginning of main
2023-06-18 15:56:37.996  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Cloud gaming: [0.037910953]
2023-06-18 15:56:37.996  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Real time: [0.15302967]
2023-06-18 15:56:37.996  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Non real time: [0.88663554]
2023-06-18 15:56:37.996  2235-2952  SemNscXgbMsL1           system_server                        D  1 sample inference time: 1.034375 msecs
2023-06-18 15:56:37.999  2235-2952  SemNscXgbL2Rt           system_server                        D  L2 RT 1 sample inference time: 2.049323 msecs
2023-06-18 15:56:38.001  2235-2952  SemNscXgbL2Nrt          system_server                        D  L2 NRT 1 sample inference time: 1.649115 msecs
2023-06-18 15:56:38.037  2235-2846  InputReader             system_server                        D  Btn_touch(1): value=1 when=44290.551010
2023-06-18 15:56:38.037  2235-2846  InputReader             system_server                        I  Touch event's action is 0x0 (id=8, t=0) [pCnt=1, s=0.566 ] when=44290.551010
2023-06-18 15:56:38.042  2235-2845  InputDispatcher         system_server                        I  Delivering touch to (6805): action: 0x0, f=0x0, d=1, '9ab2668', t=1 
2023-06-18 15:56:38.042  2235-2388  GestureDetector         system_server                        I  obtain mCurrentDownEvent. id: 1024224024 caller: com.android.server.wm.SystemGesturesPointerEventListener.onPointerEvent:203 com.android.server.wm.PointerEventDispatcher.onInputEvent:53 android.view.InputEventReceiver.dispatchInputEvent:285 
--------- beginning of system
2023-06-18 15:56:38.042  6805-6805  ViewRootIm...ngOverlay] com.eightbit.samsprung:SamSprung     I  ViewPostIme pointer 0
2023-06-18 15:56:38.044  2235-2845  PowerManagerService     system_server                        D  UserActivityStateListenerState: 1
2023-06-18 15:56:38.044  6805-6805  GestureDetector         com.eightbit.samsprung:SamSprung     I  obtain mCurrentDownEvent. id: 1024224024 caller: com.eightbit.samsprung.SamSprungOverlay$$ExternalSyntheticLambda14.onTouch:17 android.view.View.dispatchTouchEvent:15540 android.view.ViewGroup.dispatchTransformedTouchEvent:3316 
2023-06-18 15:56:38.046  6164-6164  SemDvfsHyPerManager     com.sec.android.sdhms                I  acquire hyper - TABoost/6164@24, type = 1090523139
2023-06-18 15:56:38.046  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]acquire(): Acquired ID : 19148790  [6164 / 24]    HINT :      list : [TABoost / 1] [TIMEOUT / 1500] 
2023-06-18 15:56:38.047  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:38.148  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:38.210  2235-2846  InputReader             system_server                        D  Btn_touch(1): value=0 when=44290.723703
2023-06-18 15:56:38.210  2235-2846  InputReader             system_server                        I  Touch event's action is 0x1 (id=8, t=0) [pCnt=1, s=] when=44290.723703
2023-06-18 15:56:38.210  2235-2845  InputDispatcher         system_server                        I  Delivering touch to (6805): action: 0x1, f=0x0, d=1, '9ab2668', t=1 
2023-06-18 15:56:38.211  2235-2388  GestureDetector         system_server                        I  obtain mCurrentMotionEventRaw. action: 1 id: 83889114
2023-06-18 15:56:38.212  6805-6805  ViewRootIm...ngOverlay] com.eightbit.samsprung:SamSprung     I  ViewPostIme pointer 1
2023-06-18 15:56:38.212  6805-6805  GestureDetector         com.eightbit.samsprung:SamSprung     I  obtain mCurrentMotionEventRaw. action: 1 id: 83889114
2023-06-18 15:56:38.214  6164-6164  SemDvfsHyPerManager     com.sec.android.sdhms                I  acquire hyper - TABoost/6164@24, type = 1090523139
2023-06-18 15:56:38.215  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]releaseLocked(): Released ID : 19148790
2023-06-18 15:56:38.215  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]acquire(): Acquired ID : 19148790  [6164 / 24]    HINT :      list : [TABoost / 1] [TIMEOUT / 1500] 
2023-06-18 15:56:38.246  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:38.342  2235-2388  GestureDetector         system_server                        I  handleMessage TAP
2023-06-18 15:56:38.345  6805-6805  GestureDetector         com.eightbit.samsprung:SamSprung     I  handleMessage TAP
2023-06-18 15:56:38.347  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:38.353  2235-4203  VibratorManagerService  system_server                        D  vibrate - uid: 10538, opPkg: com.eightbit.samsprung, effect: Mono{mEffect=Composed{segments=[Prebaked{effect=CLICK, strength=MEDIUM, fallback=true}], repeat=-1, mMagnitudeType=TYPE_EXTRA}}, attrs: VibrationAttributes: Usage=UNKNOWN Audio Usage= USAGE_UNKNOWN tags= Flags=0, reason: null, token: android.os.BinderProxy@430337
2023-06-18 15:56:38.353  2235-4203  VibratorManagerService  system_server                        D  converted usage = TOUCH(18)
2023-06-18 15:56:38.353  2235-4203  VibratorManagerService  system_server                        D  Starting vibrate for vibration  551
2023-06-18 15:56:38.357  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  V_2_0: Vibrator perform Effect : 0 EffectStrength : 1
2023-06-18 15:56:38.358  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: setPerformAmplitude : 6700
2023-06-18 15:56:38.358  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: [preparePerform]amplitude: 6700
2023-06-18 15:56:38.358  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: [preparePerform]CLICK
2023-06-18 15:56:38.358  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: setAmplitude : INTENSITY 6700
2023-06-18 15:56:38.358  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: >> writeNode node:/timed_output/vibrator/intensity val:6700
2023-06-18 15:56:38.359  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: << writeNode node OK : 5
2023-06-18 15:56:38.360  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: do_on for timeoutMs: 7
2023-06-18 15:56:38.360  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: >> writeNode node:/timed_output/vibrator/enable val:7
2023-06-18 15:56:38.361  2235-2408  AppOps                  system_server                        I  commitUidPendingStateLocked() :: UID - 10227, 400 > 500, 15 > 15
2023-06-18 15:56:38.362  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: << writeNode node OK : 2
2023-06-18 15:56:38.362  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  V_2_0: Vibrator perform timeMS return: 17
2023-06-18 15:56:38.362  1479-1479  SSC_DAEMON              factory.ssc                          I  physical_sensor_test_req_msg Sensor type :0, Msg type:3
2023-06-18 15:56:38.363  1479-1479  SSC_DAEMON              factory.ssc                          I  physical_sensor_test_req_msg Sensor type :6, Msg type:3
2023-06-18 15:56:38.367  1396-5125  libprotobuf-native      and....sensors@2.1-service.multihal  E  [libprotobuf ERROR external/protobuf/src/google/protobuf/message_lite.cc:123] Can't parse message of type "sns_std_sensor_event" because it is missing required fields: (cannot determine missing fields for lite message)
2023-06-18 15:56:38.368  1479-2744  SSC_DAEMON              factory.ssc                          I  accel_motor_state_event: 1, 1
2023-06-18 15:56:38.368  1479-2744  SSC_DAEMON              factory.ssc                          I  SBOM motor: 1
2023-06-18 15:56:38.368  1479-2744  SSC_DAEMON              factory.ssc                          I  sub_accel_motor_state_event: 1, 1
2023-06-18 15:56:38.379  2235-24163 VibratorManagerService  system_server                        D  Vibration 551 on vibrator 0 complete, notifying thread
2023-06-18 15:56:38.380  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  V_2_0: Vibrator off
2023-06-18 15:56:38.380  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: do_off
2023-06-18 15:56:38.380  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: >> writeNode node:/timed_output/vibrator/enable val:0
2023-06-18 15:56:38.382  1455-1455  SecVibrato...-AIDL-CORE ven...ung.hardware.vibrator-service  E  HW_API: << writeNode node OK : 2
2023-06-18 15:56:38.382  2235-2798  VibratorManagerService  system_server                        D  Vibration 551 finished with status FINISHED
2023-06-18 15:56:38.382  2235-2798  VibratorManagerService  system_server                        D  Reporting vibration 551 finished with status FINISHED
2023-06-18 15:56:38.382  2235-2798  VibratorManagerService  system_server                        D  Vibration for uid=10538 and with attrs=VibrationAttributes: Usage=TOUCH Audio Usage= USAGE_UNKNOWN tags= Flags=0 ended with status FINISHED
2023-06-18 15:56:38.383  2235-2798  VibratorManagerService  system_server                        D  VibrationThread released after finished vibration
2023-06-18 15:56:38.383  2235-2798  VibratorManagerService  system_server                        D  Processing VibrationThread released callback
2023-06-18 15:56:38.425  3066-10075 AODCommonSettingsUtils  com.android.systemui                 I  addLogText
2023-06-18 15:56:38.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:38.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:38.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:38.447  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:38.502  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Cloud gaming: [0.034194093]
2023-06-18 15:56:38.502  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Real time: [0.16628318]
2023-06-18 15:56:38.502  2235-2952  SemNscXgbMsL1           system_server                        D  Probability - Non real time: [0.8880953]
2023-06-18 15:56:38.502  2235-2952  SemNscXgbMsL1           system_server                        D  1 sample inference time: 0.618594 msecs
2023-06-18 15:56:38.503  2235-2952  SemNscXgbL2Rt           system_server                        D  L2 RT 1 sample inference time: 0.894792 msecs
2023-06-18 15:56:38.504  2235-2952  SemNscXgbL2Nrt          system_server                        D  L2 NRT 1 sample inference time: 0.726563 msecs
2023-06-18 15:56:38.559  1061-1061  io_stats                iod                                  D  !@   8,0 r 1839951 68764885 w 1192072 18368484 d 388793 20324060 f 0 113535 iot 1557768 0 th 0 0 0 pt 0 inp 0 0 44291.074
2023-06-18 15:56:38.559  1061-1061  io_stats                iod                                  D  !@ Write_top(KB): kworker/u16:14(3972) 1144 f2fs_ckpt-254:5(1141) 168 com.viber.voip(22646) 128
2023-06-18 15:56:38.639  3066-3066  SecPowerUI.Notification com.android.systemui                 D  showChargingNotice oldChargingType : 11 / currentChargingType : 11, oldChargingTime : 9384000 / mChargingTime : 9384000
2023-06-18 15:56:38.639  3066-3066  SecPowerUI.Notification com.android.systemui                 D  There is no change about charging status, so return!
2023-06-18 15:56:38.827  6805-24184 OpenGLRenderer          com.eightbit.samsprung:SamSprung     D  setSurface called with nullptr
2023-06-18 15:56:38.828  6805-24184 OpenGLRenderer          com.eightbit.samsprung:SamSprung     D  setSurface called with nullptr
2023-06-18 15:56:38.828  6805-24184 OpenGLRenderer          com.eightbit.samsprung:SamSprung     D  setSurface called with nullptr
2023-06-18 15:56:38.828  6805-24184 OpenGLRenderer          com.eightbit.samsprung:SamSprung     D  setSurface called with nullptr
2023-06-18 15:56:38.952  2235-2952  SemWifiServiceDetector  system_server                        I  Traff. tracking stopped
2023-06-18 15:56:39.243  6805-13288 Applicatio...ageManager com.eightbit.samsprung:SamSprung     D  we has com.samsung.android.calendar class. reuse it 
2023-06-18 15:56:39.244  6805-13288 LiveIconLoader          com.eightbit.samsprung:SamSprung     I  [Calendar] LiveIcon - stdIconSize : 48 , targetIconSize : 48
2023-06-18 15:56:39.246  6805-13288 LiveIconLoader          com.eightbit.samsprung:SamSprung     W  System font not found, reverting to Roboto.
2023-06-18 15:56:39.250  6805-13288 LiveIconLoader          com.eightbit.samsprung:SamSprung     I  [Calendar] getLiveIcon - Date: 18
2023-06-18 15:56:39.251  6805-13288 AppIconSolution         com.eightbit.samsprung:SamSprung     I  load= live icon for com.samsung.android.calendar, from overlay = false
2023-06-18 15:56:39.264  6805-6805  Choreographer           com.eightbit.samsprung:SamSprung     I  Skipped 52 frames!  The application may be doing too much work on its main thread.
2023-06-18 15:56:39.276  6805-14812 OpenGLRenderer          com.eightbit.samsprung:SamSprung     I  Davey! duration=897ms; Flags=0, FrameTimelineVsyncId=21760252, IntendedVsync=44290884392250, Vsync=44290884392250, InputEventId=83889114, HandleInputStart=44290885099769, AnimationStart=44290885100707, PerformTraversalsStart=44290887108363, DrawStart=44291760317061, FrameDeadline=44290901058916, FrameInterval=44290885094144, FrameStartTime=16724612, SyncQueued=44291778562425, SyncStart=44291779016956, IssueDrawCommandsStart=44291779420863, SwapBuffers=44291782033154, FrameCompleted=44291782706436, DequeueBufferDuration=513854, QueueBufferDuration=237291, GpuCompleted=44291782623883, SwapBuffersCompleted=44291782706436, DisplayPresentTime=0, CommandSubmissionCompleted=44291782033154, 
2023-06-18 15:56:39.292  6805-14812 OpenGLRenderer          com.eightbit.samsprung:SamSprung     I  Davey! duration=879ms; Flags=0, FrameTimelineVsyncId=21760274, IntendedVsync=44290901115602, Vsync=44291770791890, InputEventId=0, HandleInputStart=44291779540081, AnimationStart=44291779540342, PerformTraversalsStart=44291779688623, DrawStart=44291779928571, FrameDeadline=44290934506812, FrameInterval=44291779491331, FrameStartTime=16724544, SyncQueued=44291780052894, SyncStart=44291782722425, IssueDrawCommandsStart=44291782844873, SwapBuffers=44291783192633, FrameCompleted=44291783700029, DequeueBufferDuration=7761, QueueBufferDuration=86302, GpuCompleted=44291783700029, SwapBuffersCompleted=44291783596019, DisplayPresentTime=0, CommandSubmissionCompleted=44291783192633, 
2023-06-18 15:56:39.338  1270-1644  TrafficStats            com...le.android.apps.youtube.music  D  tagSocket(118) with statsTag=0xffffffff, statsUid=-1
2023-06-18 15:56:39.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:39.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:39.443  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:39.564  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  (REDACTED) FastPairScanner: isScreenOn=%s, isLocationEnabled=%s, disableLocationRequirement=%s, isDiscoveryScanningEnabled=%s, during24GhzWifiWarmingUpPeriod=%s
2023-06-18 15:56:39.564  4643-8257  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.564  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  (REDACTED) FastPairScanner: eventType=%s, intReq=%s, scanning=%s, scanAllowed=%s, bleEnabled=%s, bleScanAvailable=%s, lockScanRate=%s, startScanningByLowPowerMode=%s
2023-06-18 15:56:39.564  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  FastPairScanner: Stopping scan [CONTEXT service_id=265 ]
2023-06-18 15:56:39.564  4643-8257  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.564  4643-8257  BluetoothLeScanner      com.google.android.gms.persistent    D  Stop Scan with callback
2023-06-18 15:56:39.566  3389-16538 ActivityAt...ionService com.android.bluetooth                D  notifyActivityAttributionInfo UID=1002 packageName=com.android.bluetooth deviceAddress=no_active_device_address
2023-06-18 15:56:39.566  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.567  1384-1384  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  D  SerialClockVote: vote for UART CLK ON
2023-06-18 15:56:39.567  3389-16538 BtGatt.ContextMap       com.android.bluetooth                D  remove() - id: 2
2023-06-18 15:56:39.567  3389-16538 BtGatt.ContextMap       com.android.bluetooth                E  remove() - removed: 2
2023-06-18 15:56:39.567  3389-16538 bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(172)] Unregister in shim layer, scanner_id:2
2023-06-18 15:56:39.568  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  (REDACTED) FastPairScanner: isScreenOn=%s, isLocationEnabled=%s, disableLocationRequirement=%s, isDiscoveryScanningEnabled=%s, during24GhzWifiWarmingUpPeriod=%s
2023-06-18 15:56:39.568  1384-1384  vendor.qti...-wake_lock and...are.bluetooth@1.0-service-qti  D  Acquire wakelock is acquired 
2023-06-18 15:56:39.568  1384-1384  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  DeviceWakeUp: Writing IBS_WAKE_IND
2023-06-18 15:56:39.568  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  (REDACTED) FastPairScanner: Start scanning internally in %s
2023-06-18 15:56:39.568  4643-8257  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.568  4643-8257  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.568  4643-8257  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.569  4643-8257  BluetoothLeScanner      com.google.android.gms.persistent    D  Start Scan with callback
2023-06-18 15:56:39.570  3389-16538 BtGatt.ContextMap       com.android.bluetooth                D  add() - appUid: 10227, appPid: 4643, appName: com.google.uid.shared
2023-06-18 15:56:39.570  3389-3798  BtGatt.GattService      com.android.bluetooth                E  [GSIM LOG]: gsimLogHandler, msg: MESSAGE_SCAN_STOP, appName: com.google.uid.shared, scannerId: 2, reportDelayMillis=0
2023-06-18 15:56:39.570  3389-16538 bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(165)] RegisterScanner in shim layer
2023-06-18 15:56:39.574  1384-3979  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  ProcessIbsCmd: Received IBS_WAKE_ACK: 0xFC
2023-06-18 15:56:39.574  1384-3979  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  ProcessIbsCmd: Signal wack_cond_
2023-06-18 15:56:39.574  1384-1384  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  D  DeviceWakeUp: Unblocked from waiting for FC, pthread_cond_timedwait ret = 0
2023-06-18 15:56:39.574  1384-3979  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  ProcessIbsCmd: Received IBS_WAKE_IND: 0xFD
2023-06-18 15:56:39.574  3389-3568  BtGatt.GattService      com.android.bluetooth                D  onScannerRegistered() - UUID=c8e50ed1-abb8-470c-bb8e-fada61c3b5e0, scannerId=2, status=0
2023-06-18 15:56:39.574  1384-3979  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  ProcessIbsCmd: Writing IBS_WAKE_ACK
2023-06-18 15:56:39.575  4643-30124 BluetoothLeScanner      com.google.android.gms.persistent    D  onScannerRegistered() - status=0 scannerId=2 mScannerId=0
2023-06-18 15:56:39.577  3389-16538 BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.578  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.578  3389-16538 ActivityAt...ionService com.android.bluetooth                D  notifyActivityAttributionInfo UID=1002 packageName=com.android.bluetooth deviceAddress=no_active_device_address
2023-06-18 15:56:39.579  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.579  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.579  3389-3798  BtGatt.GattService      com.android.bluetooth                E  [GSIM LOG]: gsimLogHandler, msg: MESSAGE_SCAN_START, appName: com.google.uid.shared, scannerId: 2, reportDelayMillis=0
2023-06-18 15:56:39.579  4643-8257  NearbyDiscovery         com.google.android.gms.persistent    I  (REDACTED) FastPairScanner: Starting %s scanning
2023-06-18 15:56:39.579  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(269)] ScanFilterEnable in shim layer
2023-06-18 15:56:39.580  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(240)] ScanFilterAdd in shim layer
2023-06-18 15:56:39.580  3389-3790  BtGatt.ScanManager      com.android.bluetooth                D  configureFilterParamter() : onFoundTimeout | onLostTimeout | onFoundCount | numOfTrackingEntries | scanFilterRssiThreshold | trackableScanFilterRssiThreshold | matchMode
2023-06-18 15:56:39.580  3389-3790  BtGatt.ScanManager      com.android.bluetooth                D  configureFilterParamter() : __________500__|________10000__|___________1__|___________________0__|___________________-128__|____________________________-128__|________1_
2023-06-18 15:56:39.580  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.603  4643-8462  Nearby                  com.google.android.gms.persistent    I  (REDACTED) [MBleClient] Start ble scanning from %s, context tag = %s
2023-06-18 15:56:39.604  4643-8462  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.604  4643-8462  BluetoothLeScanner      com.google.android.gms.persistent    D  Stop Scan with callback
2023-06-18 15:56:39.607  3389-16538 ActivityAt...ionService com.android.bluetooth                D  notifyActivityAttributionInfo UID=1002 packageName=com.android.bluetooth deviceAddress=no_active_device_address
2023-06-18 15:56:39.607  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.609  3389-16538 BtGatt.ContextMap       com.android.bluetooth                D  remove() - id: 1
2023-06-18 15:56:39.609  3389-16538 BtGatt.ContextMap       com.android.bluetooth                E  remove() - removed: 1
2023-06-18 15:56:39.609  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.609  3389-16538 bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(172)] Unregister in shim layer, scanner_id:1
2023-06-18 15:56:39.611  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.611  4643-8462  Nearby                  com.google.android.gms.persistent    I  (REDACTED) [MBleClient] M hardware scan: %d clients, scanMode= %s, filters size = %s
2023-06-18 15:56:39.611  4643-8462  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.611  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.611  4643-8462  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.611  4643-8462  BluetoothAdapter        com.google.android.gms.persistent    I  STATE_ON
2023-06-18 15:56:39.612  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(198)] ScanFilterParamSetup in shim layer
2023-06-18 15:56:39.613  4643-8462  BluetoothLeScanner      com.google.android.gms.persistent    D  Start Scan with callback
2023-06-18 15:56:39.613  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(178)] Scan in shim layer stopped
2023-06-18 15:56:39.614  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(312)] SetScanParameters with scanning phys in shim layer: phy_cnt=1
2023-06-18 15:56:39.614  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(318)] SetScanParameters: scan_type=1, scan_interval=4992, scan_window=256
2023-06-18 15:56:39.614  3389-3790  bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(178)] Scan in shim layer started
2023-06-18 15:56:39.614  3389-3810  bt_stack                com.android.bluetooth                W  le_address_manager :: scan() - Unregister
2023-06-18 15:56:39.614  3389-3810  bt_stack                com.android.bluetooth                W  Stop LeSetExtendedScan
2023-06-18 15:56:39.614  3389-3810  bt_stack                com.android.bluetooth                W  Start LeSetExtendedScan
2023-06-18 15:56:39.614  3389-3810  bt_stack                com.android.bluetooth                W  register_client :: client registered!!!
2023-06-18 15:56:39.616  3389-16538 BtGatt.ContextMap       com.android.bluetooth                D  add() - appUid: 10227, appPid: 4643, appName: com.google.uid.shared
2023-06-18 15:56:39.616  3389-16538 bt_stack                com.android.bluetooth                I  [INFO:le_scanning_manager.cc(165)] RegisterScanner in shim layer
2023-06-18 15:56:39.616  3389-3798  BtGatt.GattService      com.android.bluetooth                E  [GSIM LOG]: gsimLogHandler, msg: MESSAGE_SCAN_STOP, appName: com.google.uid.shared, scannerId: 1, reportDelayMillis=0
2023-06-18 15:56:39.618  3389-3568  BtGatt.GattService      com.android.bluetooth                D  onScannerRegistered() - UUID=0d616f0b-ce53-4754-aa6a-8d36f81faf74, scannerId=1, status=0
2023-06-18 15:56:39.619  4643-30124 BluetoothLeScanner      com.google.android.gms.persistent    D  onScannerRegistered() - status=0 scannerId=1 mScannerId=0
2023-06-18 15:56:39.621  3389-16538 BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.622  3389-16538 ActivityAt...ionService com.android.bluetooth                D  notifyActivityAttributionInfo UID=1002 packageName=com.android.bluetooth deviceAddress=no_active_device_address
2023-06-18 15:56:39.622  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.624  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.624  3389-3790  BluetoothAdapter        com.android.bluetooth                I  STATE_ON
2023-06-18 15:56:39.625  3389-3798  BtGatt.GattService      com.android.bluetooth                E  [GSIM LOG]: gsimLogHandler, msg: MESSAGE_SCAN_START, appName: com.google.uid.shared, scannerId: 1, reportDelayMillis=0
2023-06-18 15:56:39.625  3389-3798  BtGatt.GattService      com.android.bluetooth                W  [GSIM LOG]: this is opportunistic scan, do not update DB
2023-06-18 15:56:39.644  2235-5243  SemWifiUsa...atsMonitor system_server                        D  onWifiUsabilityStats - seqNum 12523, isSameBssidAndFreq true
2023-06-18 15:56:39.649  2235-2954  SemWifiLin...ityMonitor system_server                        I  Link Qos Query: 0,012 ms / 650,000 Mbps (650 / 0,000 / 1,000)
2023-06-18 15:56:39.649  3066-3066  NetworkCon...Controller com.android.systemui                 D  notifyListener: visible=true, connected=true, inetCondition=1, isDefault=true, wifiTestReported=true, receivedInetCondition=-1, hideDuringMobileSwitching=false
2023-06-18 15:56:39.665  1384-3979  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  ProcessIbsCmd: Received IBS_SLEEP_IND: 0xFE
2023-06-18 15:56:39.716  1405-13532 HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]releaseLocked(): Released ID : 19148790
2023-06-18 15:56:40.083  2235-2846  InputReader             system_server                        D  Btn_touch(1): value=1 when=44292.598326
2023-06-18 15:56:40.084  2235-2846  InputReader             system_server                        I  Touch event's action is 0x0 (id=8, t=0) [pCnt=1, s=0.567 ] when=44292.598326
2023-06-18 15:56:40.084  2235-2845  InputDispatcher         system_server                        I  Delivering touch to (6805): action: 0x0, f=0x0, d=1, '9ab2668', t=1 
2023-06-18 15:56:40.085  2235-2845  PowerManagerService     system_server                        D  [api] userActivityFromNative : 4050 (event: 2 flags: 0 displayId: 1) eventTime = 44292598
2023-06-18 15:56:40.085  6805-6805  ViewRootIm...ngOverlay] com.eightbit.samsprung:SamSprung     I  ViewPostIme pointer 0
2023-06-18 15:56:40.085  2235-2388  GestureDetector         system_server                        I  obtain mCurrentDownEvent. id: 196707996 caller: com.android.server.wm.SystemGesturesPointerEventListener.onPointerEvent:203 com.android.server.wm.PointerEventDispatcher.onInputEvent:53 android.view.InputEventReceiver.dispatchInputEvent:285 
2023-06-18 15:56:40.089  6164-6164  SemDvfsHyPerManager     com.sec.android.sdhms                I  acquire hyper - TABoost/6164@24, type = 1090523139
2023-06-18 15:56:40.089  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]acquire(): Acquired ID : 19148790  [6164 / 24]    HINT :      list : [TABoost / 1] [TIMEOUT / 1500] 
2023-06-18 15:56:40.090  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:40.189  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:40.256  2235-2846  InputReader             system_server                        D  Btn_touch(1): value=0 when=44292.771631
2023-06-18 15:56:40.257  2235-2846  InputReader             system_server                        I  Touch event's action is 0x1 (id=8, t=0) [pCnt=1, s=] when=44292.771631
2023-06-18 15:56:40.257  2235-2845  InputDispatcher         system_server                        I  Delivering touch to (6805): action: 0x1, f=0x0, d=1, '9ab2668', t=1 
2023-06-18 15:56:40.258  6805-6805  ViewRootIm...ngOverlay] com.eightbit.samsprung:SamSprung     I  ViewPostIme pointer 1
2023-06-18 15:56:40.258  2235-2388  GestureDetector         system_server                        I  obtain mCurrentMotionEventRaw. action: 1 id: 446753490
2023-06-18 15:56:40.261  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]releaseLocked(): Released ID : 19148790
2023-06-18 15:56:40.260  6164-6164  SemDvfsHyPerManager     com.sec.android.sdhms                I  acquire hyper - TABoost/6164@24, type = 1090523139
2023-06-18 15:56:40.262  1405-1429  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]acquire(): Acquired ID : 19148790  [6164 / 24]    HINT :      list : [TABoost / 1] [TIMEOUT / 1500] 
2023-06-18 15:56:40.281  6805-6805  AssistStructure         com.eightbit.samsprung:SamSprung     I  Flattened final assist data: 4044 bytes, containing 1 windows, 31 views
2023-06-18 15:56:40.282  2235-3887  AutofillSession         system_server                        D  createPendingIntent for request 39
2023-06-18 15:56:40.282  2235-3887  ContentCap...serService system_server                        D  Notified activity assist data for activity: Token{4e67945 ActivityRecord{7844090 u0 com.eightbit.samsprung/.SamSprungOverlay} t571}} without a session Id
2023-06-18 15:56:40.289  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:40.296  5798-5798  SSS:Conten...ureService com...sung.android.smartsuggestions  I  onActivitySnapshot()
2023-06-18 15:56:40.305  2486-10650 [PASS][SPA...inProvider com.samsung.android.samsungpass      I  query called from com.samsung.android.samsungpassautofill
2023-06-18 15:56:40.310 11422-11422 [SPAF]AutofillRepo      com...g.android.samsungpassautofill  I  has DAL field in Pass DB
2023-06-18 15:56:40.311  2486-10650 [PASS][SPA...inProvider com.samsung.android.samsungpass      I  query called from com.samsung.android.samsungpassautofill
2023-06-18 15:56:40.315  2235-3887  AssistStructure         system_server                        I  Flattened final assist data: 4064 bytes, containing 1 windows, 31 views
2023-06-18 15:56:40.316 11422-11422 [SPAF]AutofillHints     com...g.android.samsungpassautofill  I  findFromEditText() / entry [search_src_text], hint [   ] => anonymous
2023-06-18 15:56:40.316 11422-11422 [SPAF]AutofillHints     com...g.android.samsungpassautofill  W  Valid autofill hint: [anonymous]
2023-06-18 15:56:40.316 11422-11422 [SPAF]MetadataCollect   com...g.android.samsungpassautofill  I  Add: 0, search_src_text
2023-06-18 15:56:40.317 11422-11422 [SPAF]AutofillHints     com...g.android.samsungpassautofill  W  Valid autofill hint: [anonymous]
2023-06-18 15:56:40.317 11422-11422 [SPAF]AutofillService   com...g.android.samsungpassautofill  E  No valid parsed autofill fields
2023-06-18 15:56:40.321  5798-5798  SSS:Augmen...illService com...sung.android.smartsuggestions  I  AugmentedAutofillService: onFillRequest()
2023-06-18 15:56:40.386  2235-2388  GestureDetector         system_server                        I  handleMessage TAP
2023-06-18 15:56:40.390  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:40.444  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:40.444  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:40.444  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:40.490  1405-1405  HYPER-HAL               ven...amsung.hardware.hyper-service  I  [ResourceManager.cpp]getResourceManager(): Resouce not supported on this device - resourceID : 0x1000
2023-06-18 15:56:40.561  1061-1061  io_stats                iod                                  D  !@   8,0 r 1840087 68773209 w 1192075 18368500 d 388793 20324060 f 0 113535 iot 1557868 0 th 0 0 0 pt 0 inp 0 0 44293.075
2023-06-18 15:56:40.625  1384-3981  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  I  DeviceSleep: TX Awake, Sending SLEEP_IND
2023-06-18 15:56:40.625  1384-3981  vendor.qti...bs_handler and...are.bluetooth@1.0-service-qti  D  SerialClockVote: vote for UART CLK OFF
2023-06-18 15:56:40.775  1384-3818  vendor.qti...-wake_lock and...are.bluetooth@1.0-service-qti  D  Release wakelock is released 
2023-06-18 15:56:40.863  2235-2800  ActivityTaskManager     system_server                        D  SafeActivityOptions created, mOriginalCallingPid=2235, mOriginalCallingUid=1000, options=ActivityOptions(191825021), mPackageName=null, mAnimationType=-1, mStartX=0, mStartY=0, mWidth=0, mHeight=0, caller=com.android.server.am.PendingIntentRecord.sendInner:471 com.android.server.am.ActivityManagerService.sendIntentSender:6573 com.android.server.am.ActivityManagerService.sendIntentSender:6562 android.app.PendingIntent.sendAndReturnResult:1029 android.app.PendingIntent.send:1000 com.android.server.alarm.AlarmManagerService$DeliveryTracker.deliverLocked:6377 com.android.server.alarm.AlarmManagerService.deliverAlarmsLocked:5218 
2023-06-18 15:56:40.866  2235-2800  ActivityManager         system_server                        D  Received BROADCAST intent 0x6e551e6 Key{broadcastIntent pkg=com.facebook.orca intent=act=WakingExecutorService.ACTION_ALARM.com.facebook.orca.Mqtt_Wakeup pkg=com.facebook.orca flags=0x4000000 u=0} requestCode=0 sent=0 from uid 1000
2023-06-18 15:56:41.364  2235-4203  SGM:GameManager         system_server                        D  identifyForegroundApp. com.eightbit.samsprung, mCurrentUserId: 0, callerUserId: 0
2023-06-18 15:56:41.364  2235-4203  SGM:PkgDataHelper       system_server                        D  getGamePkgData(). com.eightbit.samsprung
2023-06-18 15:56:41.365 12492-12630 PolicyManager           com.samsung.cmh:CMH                  I  [#CMH#] Is foreground game app: false
2023-06-18 15:56:41.446  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:41.446  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:41.446  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:41.765  1405-13536 HYPER-HAL               ven...amsung.hardware.hyper-service  I  [RequestManager.cpp]releaseLocked(): Released ID : 19148790
2023-06-18 15:56:42.447  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:42.447  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:42.447  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:42.566  1061-1061  io_stats                iod                                  D  !@   8,0 r 1840087 68773209 w 1192090 18368980 d 388793 20324060 f 0 113535 iot 1557876 0 th 0 0 0 pt 0 inp 0 0 44295.079
2023-06-18 15:56:42.566  1061-1061  io_stats                iod                                  D  !@ Read_top(KB): prung:SamSprung(6805) 8324
2023-06-18 15:56:42.566  1061-1061  io_stats                iod                                  D  !@ Write_top(KB): kworker/u16:1(10085) 480 kworker/u16:14(3972) 16
2023-06-18 15:56:42.567  9978-9978  adbd                    adbd                                 W  timeout expired while flushing socket, closing
2023-06-18 15:56:42.689  2235-5243  SemWifiUsa...atsMonitor system_server                        D  onWifiUsabilityStats - seqNum 12524, isSameBssidAndFreq true
2023-06-18 15:56:42.690  2235-2954  SemWifiLin...ityMonitor system_server                        I  Link Qos Query: 0,012 ms / 671,479 Mbps (702 / 0,001 / 1,000)
2023-06-18 15:56:42.815  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:169, auto_rotation_debug_2 mode,255, type,1,1 acc,-2.812,3.417,9.227, ar,0, ver,10
2023-06-18 15:56:43.042  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:151, auto_rotation_event_2 event,255, duration,486, ts,80192535693062, ts_diff,3524410
2023-06-18 15:56:43.042  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:154, auto_rotation_debug_2 acc,0.584,2.845,9.509, ar,0, ver,10
2023-06-18 15:56:43.046  3066-3066  AppController           com.android.systemui                 D  AppControlSensorListener.onSensorChanged, Rotation: -1
2023-06-18 15:56:43.242  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:151, auto_rotation_event_2 event,0, duration,0, ts,80192573851240, ts_diff,165834565
2023-06-18 15:56:43.243  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:154, auto_rotation_debug_2 acc,0.821,3.147,9.174, ar,0, ver,10
2023-06-18 15:56:43.243  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:151, auto_rotation_event_2 event,255, duration,0, ts,80192591553635, ts_diff,148796077
2023-06-18 15:56:43.243  1396-13423 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_client_event:154, auto_rotation_debug_2 acc,0.768,3.070,9.174, ar,0, ver,10
2023-06-18 15:56:43.245  3066-3066  AppController           com.android.systemui                 D  AppControlSensorListener.onSensorChanged, Rotation: 0
2023-06-18 15:56:43.245  3066-3066  AppController           com.android.systemui                 D  AppControlSensorListener.onSensorChanged, Rotation: -1
2023-06-18 15:56:43.262  2235-2445  PowerManagerService     system_server                        D  UserActivityStateListenerState: 0
2023-06-18 15:56:43.268  7458-7458  HBD                     com.samsung.android.honeyboard       I  j defaultInputMethod :  com.eightbit.samsprung.ime/.input.SamSprungInput
2023-06-18 15:56:43.268  7458-7458  HBD                     com.samsung.android.honeyboard       I  e onChange propertyId=4 
2023-06-18 15:56:43.278  2235-2775  ProcessStats            system_server                        W  Tracking association SourceState{39abfe8 system/1000 BTopFgs #156001} whose proc state 2 is better than process ProcessState{43cc03f com.eightbit.samsprung.ime:SamSprung/10539 pkg=com.eightbit.samsprung.ime} proc state 3 (205 skipped)
2023-06-18 15:56:43.416  2235-4052  FreecessController      system_server                        D  com.samsung.android.app.notes(state: Freezeable -> Frozen, Reason: provider)
2023-06-18 15:56:43.419  2235-4052  FreecessController      system_server                        D  FZ : com.samsung.android.app.notes(10936)
2023-06-18 15:56:43.426  2235-4052  FreecessController      system_server                        D  FZ : com.samsung.android.app.notes(0), reason: Bg
2023-06-18 15:56:43.432  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10280 , disable= true
2023-06-18 15:56:43.432  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10280
2023-06-18 15:56:43.433  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10280]=true
2023-06-18 15:56:43.449  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  pcf : 0x1012, 0 ,2 ,0 ,0 ,0 ,2, 7.0.0.1
2023-06-18 15:56:43.449  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  thread id : 2, preenroll_flag : 0, nd cnt : 1, cso : 0, et : 0
2023-06-18 15:56:43.449  1941-2998  bauth_FPBAuthService    ven...trics.fingerprint@3.0-service  I  FPBAuthService, 11419
2023-06-18 15:56:43.456  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  set_config:61, sample_period_ns is adjusted to 200000000 based on min/max delay_ns
2023-06-18 15:56:43.456  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  batch:503, com.samsung.sensor.light_cct/1511, period=200000000, max_latency=0 request completed
2023-06-18 15:56:43.456  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  activate:419, com.samsung.sensor.light_cct/1511 en=1
2023-06-18 15:56:43.457  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  get_qmi_debug_flag:239, support_qmi_debug : false
2023-06-18 15:56:43.459  1368-1593  minksocket              ssgtzd                               E  MinkIPC_QRTR_Service: client with node 1 port 6f5c went down
2023-06-18 15:56:43.460  1368-1593  minksocket              ssgtzd                               E  MinkIPC_QRTR_Service: client with node 1 port 6f5d went down
2023-06-18 15:56:43.463  1368-1593  minksocket              ssgtzd                               E  MinkIPC_QRTR_Service: client with node 1 port 6f5e went down
2023-06-18 15:56:43.465  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  send_sync_sensor_request:448, send sync request
2023-06-18 15:56:43.465  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  send_sync_sensor_request:474, wait for notification of response
2023-06-18 15:56:43.467  1368-1593  minksocket              ssgtzd                               E  MinkIPC_QRTR_Service: client with node 1 port 6f5f went down
2023-06-18 15:56:43.468  1396-13544 sensors-hal             and....sensors@2.1-service.multihal  I  ssc_conn_resp_cb:840, resp_value = 0
2023-06-18 15:56:43.468  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  send_sync_sensor_request:480, takes 3 ms to receive the response with 0
2023-06-18 15:56:43.468  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  activate:437, com.samsung.sensor.light_cct/1511 en=1 completed
2023-06-18 15:56:43.469  2235-3319  SensorManager           system_server                        D  registerListener :: 1511, STK33915 Light CCT  Non-wakeup, 200000, 0, com.samsung.android.displaysolution.EyeComfortSolutionService$2@d003895
2023-06-18 15:56:43.535  1396-13544 sensors-hal             and....sensors@2.1-service.multihal  I  handle_indication_realtime:467,  SCHED_FIFO(10) for qmi_cbk
2023-06-18 15:56:43.536  1396-13544 sensors-hal             and....sensors@2.1-service.multihal  I  handle_sns_std_sensor_event:89, light_cct: ts = 80193025330823 ns; value = [0, 0, 0, 0, 0, 0, 1, 0, 0, -1]
2023-06-18 15:56:43.541  2235-3319  SensorService           system_server                        D  Calling activate off 1511
2023-06-18 15:56:43.542  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  activate:419, com.samsung.sensor.light_cct/1511 en=0
2023-06-18 15:56:43.543  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  deactivate:380, disconnect qmi to disable sensor
2023-06-18 15:56:43.543  1396-1396  sensors-hal             and....sensors@2.1-service.multihal  I  activate:437, com.samsung.sensor.light_cct/1511 en=0 completed
2023-06-18 15:56:43.544  2235-3319  SensorManager           system_server                        D  unregisterListener :: com.samsung.android.displaysolution.EyeComfortSolutionService$2@d003895
2023-06-18 15:56:43.544  1368-1593  minksocket              ssgtzd                               E  MinkIPC_QRTR_Service: client with node 1 port 6f60 went down
2023-06-18 15:56:43.567  2235-4052  FreecessController      system_server                        D  com.google.android.apps.youtube.music(state: Freezeable -> Frozen, Reason: broadcast)
2023-06-18 15:56:43.570  2235-4052  FreecessController      system_server                        D  FZ : com.google.android.apps.youtube.music(1270)
2023-06-18 15:56:43.573  2235-4052  FreecessController      system_server                        D  FZ : com.google.android.apps.youtube.music(0), reason: Bg
2023-06-18 15:56:43.581  2235-4052  FreecessController      system_server                        D  com.ogden.memo(state: Freezeable -> Frozen, Reason: broadcast)
2023-06-18 15:56:43.582  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10276 , disable= true
2023-06-18 15:56:43.582  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10276
2023-06-18 15:56:43.582  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10276]=true
2023-06-18 15:56:43.584  2235-4052  FreecessController      system_server                        D  FZ : com.ogden.memo(15728)
2023-06-18 15:56:43.586  2235-4052  FreecessController      system_server                        D  FZ : com.ogden.memo(0), reason: Bg
2023-06-18 15:56:43.593  2235-4052  FreecessController      system_server                        D  com.viber.voip(state: Freezeable -> Frozen, Reason: broadcast)
2023-06-18 15:56:43.593  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10422 , disable= true
2023-06-18 15:56:43.593  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10422
2023-06-18 15:56:43.594  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10422]=true
2023-06-18 15:56:43.598  2235-4052  FreecessController      system_server                        D  FZ : com.viber.voip(22646)
2023-06-18 15:56:43.600  2235-4052  FreecessController      system_server                        D  FZ : com.viber.voip(0), reason: Bg
2023-06-18 15:56:43.606  2235-4052  FreecessController      system_server                        D  com.whatsapp(state: Freezeable -> Frozen, Reason: broadcast)
2023-06-18 15:56:43.606  2235-4052  FreecessController      system_server                        D  com.instagram.android(state: Freezeable -> Frozen, Reason: broadcast)
2023-06-18 15:56:43.606  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10333 , disable= true
2023-06-18 15:56:43.606  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10333
2023-06-18 15:56:43.606  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10333]=true
2023-06-18 15:56:43.607  2235-4052  FreecessController      system_server                        D  FZ : com.whatsapp(12330)
2023-06-18 15:56:43.608  2235-4052  FreecessController      system_server                        D  FZ : com.whatsapp(0), reason: Bg
2023-06-18 15:56:43.614  2235-4052  FreecessController      system_server                        D  FZ : com.instagram.android(12514)
2023-06-18 15:56:43.616  2235-4052  FreecessController      system_server                        D  FZ : com.instagram.android(0), reason: Bg
2023-06-18 15:56:43.620  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10339 , disable= true
2023-06-18 15:56:43.621  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10339
2023-06-18 15:56:43.621  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10339]=true
2023-06-18 15:56:43.621  2235-4052  PowerManagerService     system_server                        I  [PWL] SetWakeLockEnableDisable uid = 10466 , disable= true
2023-06-18 15:56:43.621  2235-4052  PowerManagerService     system_server                        I  [PWL] can not change uid =  10466
2023-06-18 15:56:43.621  2235-4052  LocationManagerService  system_server                        W  onFreezeStateChanged, uid[10466]=true
2023-06-18 15:56:43.847  1391-1427  BatteryDump             and...re.health@2.1-service-samsung  D  wait_for_battery_event : occurred : change@/devices/platform/samsung_mobile_device/samsung_mobile_device:battery/power_supply/battery 
2023-06-18 15:56:43.850  2235-2906  DeviceStor...torService system_server                        W  updateBroadcasts(/data) oldLevel:0, newLevel:0, seq:1
2023-06-18 15:56:43.851  1391-1427  BatteryDump             and...re.health@2.1-service-samsung  E  !@new_battery_dump : 3946,-316,475,550,60,307,Charging,Normal,Good,USB,1,Normal,0,0,11333,0,0000,0000,00000000,3888,3932,62,125,3,FG LOG
2023-06-18 15:56:43.851  2235-2906  DeviceStor...torService system_server                        D  Available File Node Number is [12718772]
2023-06-18 15:56:43.851  2235-2906  DeviceStor...torService system_server                        W  updateBroadcasts_filenode(/data) fn_oldLevel:0, fn_newLevel:0, seq:1
2023-06-18 15:56:43.855  1391-1391  android.ha...ce-samsung and...re.health@2.1-service-samsung  I  updateLrpSysfs: write: 327
2023-06-18 15:56:43.856  2235-2929  BatteryService          system_server                        D  Sending ACTION_BATTERY_CHANGED: level:60, status:2, health:2, ac:false, usb:true, wireless:false, pogo:false, misc:0x10000, charge_type:1, charger_type:0, voltage:3946, temperature:307, online:4, current_avg:-24, ps:false, hvc:false, capacity:280000, current_event:0x10000, current_now:-316, mcc:0, mcv:0, cc:1985076, present:true, scale:100, technology:Li-ion
2023-06-18 15:56:43.858  2235-2829  MotionReco...ionService system_server                        D  [SAR]  onReceive : ACTION_BATTERY_CHANGED, Plugged: 2
2023-06-18 15:56:43.859  2235-2235  PhoneWindowManagerExt   system_server                        D  ACTION_BATTERY_CHANGED - Level=60, status=2
2023-06-18 15:56:43.860  2235-2987  DeviceBatt...nfoService system_server                        I  action: android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.860  2235-2987  DeviceBatt...nfoService system_server                        I  phone battery level: 60
2023-06-18 15:56:43.860  2235-2987  DeviceBatt...nfoService system_server                        I  phone battery status: 2
2023-06-18 15:56:43.860  2235-2235  AdaptiveBr...delBuilder system_server                        D  Received android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.860  2235-2235  SemWifiApB...stReceiver system_server                        D  Received : android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.862  3047-3047  SemBigData...Controller com.android.phone                    D  Intent: android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.862  3047-3047  SemBigData...Controller com.android.phone                    D  Intent: android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.862  3066-3066  PowerUI                 com.android.systemui                 I  mBootCompleted = true |  mFullyConnected = true
2023-06-18 15:56:43.863  3066-3066  PowerUI                 com.android.systemui                 I  BATTERY_HEALTH_CHECK extraHealth=2 mBatteryMiscEvent=65536
2023-06-18 15:56:43.863  3066-3066  PowerUI                 com.android.systemui                 W  Plug reason
2023-06-18 15:56:43.863  3066-5681  SecPowerUI.Notification com.android.systemui                 D  dismissing low battery warning: level=60
2023-06-18 15:56:43.864  3066-3066  PowerUI                 com.android.systemui                 W  Skip charging animation - by plug status
2023-06-18 15:56:43.864  3066-5681  SecPowerUI.Notification com.android.systemui                 D  updateNotification mWarning=false mPlaySound=false
2023-06-18 15:56:43.864  3066-3066  PowerUI                 com.android.systemui                 D  priorPlugType = 2 mPlugType = 2 priorBatteryStatus = 2 mBatteryStatus = 2
2023-06-18 15:56:43.864  3066-5681  SecPowerUI.Notification com.android.systemui                 D  SecBatteryStatsSnapshot{batteryLevel=60, currentBatteryMode=0, chargingTime=9384000, chargingType=11, batteryHealth=2, automaticTestMode=false, isHiccupState=false}
2023-06-18 15:56:43.864  3066-3066  PowerUI                 com.android.systemui                 W  Plug reason
2023-06-18 15:56:43.864  3066-3066  PowerUI                 com.android.systemui                 W  Skip charging sound - by plug status
2023-06-18 15:56:43.864  3066-3066  PowerUI                 com.android.systemui                 I  oldMisalign : false, curMisalign : false
2023-06-18 15:56:43.865  3066-5681  SecPowerUI.Notification com.android.systemui                 D  1. restoreScreenTimeOut - saved_value : :
2023-06-18 15:56:43.865  3066-5681  SecPowerUI.Notification com.android.systemui                 E  no saved value, so we do nothing !!
2023-06-18 15:56:43.866  3066-3066  SecPowerUI.Notification com.android.systemui                 D  showChargingNotice oldChargingType : 11 / currentChargingType : 11, oldChargingTime : 9384000 / mChargingTime : 9384000
2023-06-18 15:56:43.866  3066-3066  SecPowerUI.Notification com.android.systemui                 D  There is no change about charging status, so return!
2023-06-18 15:56:43.866  3066-3066  PowerUI                 com.android.systemui                 D  Battery swelling mode - priorBatterySwellingMode = 0 mBatterySwellingMode = 0 mBatteryStatus = 2
2023-06-18 15:56:43.866  3066-3066  PowerUI                 com.android.systemui                 I  mBatteryMiscEvent = 65536
2023-06-18 15:56:43.867  3066-3066  KeyguardUpdateMonitor   com.android.systemui                 D  received broadcast android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.867  2235-2408  BaseRestrictionMgr      system_server                        D  Package: com.google.android.apps.youtube.music, userid: 0, hostingType: broadcast is Restricted by policy: 4 caller is: null
2023-06-18 15:56:43.868  2235-2408  BroadcastQueue          system_server                        W  intent:Intent { act=android.intent.action.BATTERY_CHANGED flg=0x60000010 (has extras) } is skipped in RestrictedPackage to ProcessRecord{66e565a 1270:com.google.android.apps.youtube.music/u0a276}
2023-06-18 15:56:43.868  3066-3066  SamsungBat...erDrawable com.android.systemui                 D  onBatteryLevelChanged isSomethingChanged: false
2023-06-18 15:56:43.868  3066-3066  SamsungBat...erDrawable com.android.systemui                 D  onBatteryLevelChanged isSomethingChanged: false
2023-06-18 15:56:43.868  3066-3066  SamsungBat...erDrawable com.android.systemui                 D  onBatteryLevelChanged isSomethingChanged: false
2023-06-18 15:56:43.869  3066-3066  SamsungBat...erDrawable com.android.systemui                 D  onBatteryLevelChanged isSomethingChanged: false
2023-06-18 15:56:43.869 10764-10764 [SCPMV2][S...rReceiver] com.samsung.android.scpm             I  onReceive: android.intent.action.BATTERY_CHANGED
2023-06-18 15:56:43.869  3066-3066  AODBatteryManager       com.android.systemui                 I  updateBatteryData: ACTION_BATTERY_CHANGED
2023-06-18 15:56:43.869  2235-2408  BaseRestrictionMgr      system_server                        D  Package: jp.co.canon.ic.cameraconnect, userid: 0, hostingType: broadcast is Restricted by policy: 4 caller is: null
2023-06-18 15:56:43.869  2235-2408  BroadcastQueue          system_server                        W  intent:Intent { act=android.intent.action.BATTERY_CHANGED flg=0x60000010 (has extras) } is skipped in RestrictedPackage to ProcessRecord{94b3a7 11173:jp.co.canon.ic.cameraconnect/u0a361}
2023-06-18 15:56:43.870  2235-2408  BaseRestrictionMgr      system_server                        D  Package: com.whatsapp, userid: 0, hostingType: broadcast is Restricted by policy: 4 caller is: null
2023-06-18 15:56:43.870  2235-2408  BroadcastQueue          system_server                        W  intent:Intent { act=android.intent.action.BATTERY_CHANGED flg=0x60000010 (has extras) } is skipped in RestrictedPackage to ProcessRecord{ad9fba5 12330:com.whatsapp/u0a339}
2023-06-18 15:56:43.870  2235-2408  BaseRestrictionMgr      system_server                        D  Package: com.instagram.android, userid: 0, hostingType: broadcast is Restricted by policy: 4 caller is: null
2023-06-18 15:56:43.870  2235-2408  BroadcastQueue          system_server                        W  intent:Intent { act=android.intent.action.BATTERY_CHANGED flg=0x60000010 (has extras) } is skipped in RestrictedPackage to ProcessRecord{a3dfca4 12514:com.instagram.android/u0a466}
2023-06-18 15:56:43.870  3066-3066  AODBatteryManager       com.android.systemui                 I  saveBatteryData : AOD BatteryData [mBatteryLevel=60, mBatteryStatus=CHARGING, mBatteryPlugType=USB, mBatteryPlugged=true, mRemainingChargeTime=9384000, mBatteryChargingType=1, mBatteryChargerType=NORMAL, mBatteryOnline=USB, mBatterySwellingMode=NONE, mBatteryProtectMode=false]
2023-06-18 15:56:43.870  3066-3066  KeyguardUpdateMonitor   com.android.systemui                 D  handleBatteryUpdate
2023-06-18 15:56:43.871 10804-10804 [SC]Batter...ceiverImpl com.samsung.android.scloud           I  onReceive: android.intent.action.BATTERY_CHANGED
```

### Guide Confirmation

- [X] I have reviewed the guide and included valid logcat content

**Comment by AbandonedCart** [comment]
> they are seemingly reshuffled randomly and icons disappear, only reappearing when launched with the phone open

This is an option you had to enable. Disabling the usage access switch will restore alphabetical order. 

More information can be found at https://samsprung.github.io/launcher/index.html under "Launcher App Settings"

**Comment by srykho22** [comment]
> Disabling the usage access switch will restore alphabetical order

I tried disabling usage access, and now the affected apps don't appear at all (even when visibility is switched on in the TooUI app). They are replaced with duplicates of apps further up the list, and tapping those duplicates launches their respective apps (so it's not a mistexture but the apps actually just not being there). Interestingly, apps that come after the missing ones alphabetically also show up after the duplicates

**Comment by AbandonedCart** [comment]
Interesting. I am unable to replicate that issue.

**Comment by srykho22** [comment]
I tried uninstalling and reinstalling the affected apps, uninstalling and reinstalling TooUI and ToolME, and clearing the cache of the aforementioned. My current workaround is enabling bixby so I can open the app while the phone is closed, and switching ToolME to enable usage access.

Is there anything further I could do to help?

**Comment by AbandonedCart** [comment]
I'll let you know, thanks. The test device is currently charging to do more testing.
